src/SonarScanner.MSBuild.Common/SonarProperties.cs

```diff
@@ -57,5 +57,7 @@ public static class SonarProperties
 
         public const string VsCoverageXmlReportsPaths = "sonar.cs.vscoveragexml.reportsPaths";
         public const string VsTestReportsPaths = "sonar.cs.vstest.reportsPaths";
+
+        public const string ClientCertPath = "clientcert.path";
```

Comment:
Can you please rename the property's value to something like sonar.* ? Thanks !

Reply:
Sure, it is done 👍